conans/client/graph/install_graph.py

```diff
@@ -454,47 +454,69 @@ def install_build_order(self):
                   "order": [[n.serialize() for n in level] for level in install_order]}
         return result
 
-    def raise_errors(self):
+    def get_missing_invalid_packages(self):
+        def analyze_package(package):
+            if package.binary == BINARY_MISSING:
+                missing.append(package)
+            elif package.binary == BINARY_INVALID:
+                invalid.append(package)
         missing, invalid = [], []
-        for ref, install_node in self._nodes.items():
-            for package in install_node.packages.values():
-                if package.binary == BINARY_MISSING:
-                    missing.append(package)
-                elif package.binary == BINARY_INVALID:
-                    invalid.append(package)
+        for _, install_node in self._nodes.items():
+            if self._order == "recipe":
+                for package in install_node.packages.values():
+                    analyze_package(package)
+            elif self._order == "configuration":
+                analyze_package(install_node)
```

Comment:
This is a different bug fix, isn't it? In these cases it makes sense to move it to its own PR, specially when so close to release date, so it can be in the release irrespective if the other feature is delayed.